### ISSUE TITLE
Unified Tracker

### ISSUE THREAD

**Issue by mm12** [jmir1/tachiyomi-mi#1403]
### Describe your suggested feature

Allow users to turn on a "unified tracker" setting. When enabled, the current tracker setup is moved to the dot menu. The user can edit the individual trackers there, the same as the current setup. 

Instead, the user sees only 1 tracker. This tracker updates each of the normal trackers for the user, automatically keeping them up to date. If one of the trackers has an error, the fields that failed to update should be red, or otherwise indicated.

Here is what the UI might look like: 
<img width="1342" alt="image" src="https://github.com/aniyomiorg/aniyomi/assets/5874051/dd3df455-e495-42ed-902c-d1d9391779c9">

The "AY" is the Aniyomi logo, shown by default if no trackers are added. Otherwise, the icons of the trackers that are included should be in a grid in that spot. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.2.4](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by sinnghabhay80** [comment]
Great idea!
Just a doubt:
If one tracker failed example, MAL failed field will be indicated, will the failed tracker also be shown?

**Comment by LittleFreak** [comment]
What's with simkl, when you use MAL too?
Aniyomi doesn't support start/end dates for SIMKL yet, so it would always throw an error by your definition.
>!Then again - SIMKL is broken in so many ways anyway... 💔😅


Besides of that good suggestion ♥️

**Comment by LittleFreak** [comment]
True, but you're ignoring my argument here.
If your suggestion applied to Anime only, then you could do the same in the Mihon (Ex-Tachiyomi) Repo, right? But you decideded to post it here, which is fine — but you'll have to think about the whole App, which includes the Anime-Functions aswell.

TL;Dr: SIMKL was just a feasable Example, which has to be kept in mind.

**Comment by LittleFreak** [comment]
> Yeah? I am saying it should support all trackers in the app. To me, it sounded like you were saying "Simkl is better than MAL, don't complain that it stops working", so i said "simkl doesn't support manga". I didn't see the comment above for some reason. 

Oh, I see. You can relax, it was just a communication problem ^^
In fact I'd like to use trackt over simkl (I despite it's lack of functionality), and don't use it for Anime, but my personal preferences won't matter here. 
Anyway...it wasn't my intention to suggest you to "just" use another service, although I get, why my previous wording might lead to that conclusion. 

> > Aniyomi doesn't support start/end dates for SIMKL yet
> 
> Yeah, and MangaUpdates doesn't support start/end dates, at all. Simply don't fill out fields from the unified tracker in the trackers that do not have them. 
> In the same way, some trackers use 1-10 rating in intervals of 1, 0.5, and 0.1. Simply ask the user to rate by 0.1 and round, unless the user overrides it.

_Im not a Dev (yet) and can only provide some input._
I see a big open vector of bugs, that will likely happen, if you just have an unified input for all trackers. As we both mentioned, there are some unsupported features in some of them, like start/end date, selected status (which already breaks the current tracking system =>#1415).

Also some user use different tracker of the same entry for a reason - even with intended different watch status. This won't work with an unified system.

**Comment by Soitora** [comment]
> > .. do the same in the Mihon (Ex-Tachiyomi) Repo, right? But you decideded to post it here, which is fine — but you'll have to ...
> 
> I also wanted to point out that Tachi is dead. Mihon is a separate project just like this is. Same dev as AZ. The reason I didn't open a issue there is 2-fold:

Hey

Not same dev as AZ btw, AZ was never involved in both Mihon or Tachiyomi, and Mihon creator was never involved with AZ.

Also, an issue about this has been long in the Tachiyomi repo: [#8459](https://git.mihon.dev/tachiyomi/tachiyomi/issues/8459), I also ported it to Mihon repo: [#390](https://github.com/mihonapp/mihon/issues/390) now.

Thanks for reminding me, and for adding on to the idea (with concept image)

**Comment by kaoru2** [comment]
Here's my proposed UI layout and colors for the unified tracker:

[![image](https://github.com/aniyomiorg/aniyomi/assets/120291360/ce2316d2-ec51-4931-a458-7e2ee522709f)](https://www.figma.com/file/8bzhvxg12uSJcmvrKtrLxh/Unified-Tracker?type=design&node-id=0%3A1&mode=design&t=eFuizs5vMarQjZAZ-1)

the red and yellow at the background of tracker's logos signify that the tracker is inaccessible(error 401 or related), and the series has not been registered to the tracker(like new series), respectively. 

Hope this helps.

**Comment by kaoru2** [comment]
> Do we need +/- buttons? and what about the end date? -> I would move the score next to the progress, and have an end date at the bottom.

Added some changes and here it is: 
![image](https://github.com/aniyomiorg/aniyomi/assets/120291360/6013e51d-cad0-4628-8996-774901eebeb5)

**Comment by AntsyLich** [comment]
@kaoru2 your mockup looks great. But have you given the thought of what to do when trackers can have different options (e.g. some trackers has their own exclusive reading status for which the other trackers doesn't have a equivalence)

**Comment by Soitora** [comment]
> @kaoru2 your mockup looks great. But have you given the thought of what to do when trackers can have different options (e.g. some trackers has their own exclusive reading status for which the other trackers doesn't have a equivalence)

I'd assume if a user opts to use something like this, in the case where it's a niche feature of a single tracker, just not let the user use the function. Or allow user to edit individual on another page, where it won't be overridden unless user changes the Unified status

**Comment by Eagle1337** [comment]
Another issue is when trackers have different chapter total counts.
